### ISSUE TITLE
Validation rules autocomplete in FormRequest "rules" method 

### ISSUE THREAD

**Issue by sautchenkov** [laravel-idea/plugin#136]
Validations rules autocomplete not working in FormRequest class "rules" method.
Example: https://monosnap.com/file/ZgOGCmfngtNtCGuEl69IZaAjl0Rgx5

Maybe it can be added? 

Thanks for your work!

**Comment by adelf** [comment]
Hi. It should work. Does it work in other places?

![image](https://user-images.githubusercontent.com/2818394/96710897-5fa34800-13a5-11eb-89fa-c399d8b95153.png)

**Comment by sautchenkov** [comment]
I checked:
1) In my Laravel 6 project all working well.
2) But in the Laravel 5.3 project working only if custom Request extends from Request, and not working if extend from FormRequest, as LaravelIdea create. 
working: 
![image](https://user-images.githubusercontent.com/1299654/96734992-c172aa80-13c3-11eb-87c1-d994d1d3d88f.png)
Not working:
https://monosnap.com/file/3AKhM5DZIm6rloZ2vh5cyJZ5vtjsTb
![image](https://user-images.githubusercontent.com/1299654/96735340-24fcd800-13c4-11eb-81e2-bed41aeb1975.png)

Maybe you not support so old Laravel version. Anyway, I found a working solution for me. Thanks.
